consensus/enclave/impl/src/lib.rs

```diff
@@ -731,7 +748,7 @@ impl ConsensusEnclave for SgxConsensusEnclave {
         key_images.sort();
 
         // Get the list of MintConfigTxs included in the block.
-        self.validate_mint_config_txs(&inputs.mint_config_txs)?;
+        self.validate_mint_config_txs(&inputs.mint_config_txs, parent_block.index, config)?;
```

Comment:
The param is named `current_block_index` but here we pass `parent_block.index`

Reply:
Good catch